src/components/comment-list-item.component.js

```diff
@@ -243,10 +269,20 @@ class CommentListItemComponent extends Component {
       (comment.body_html && comment.body_html !== '') ||
       (comment.body && comment.body !== '');
 
+    const commentActionSheetOptions = [
+      translate('issue.comment.editAction', language),
+      translate('issue.comment.deleteAction', language),
+    ];
+
+    const isActionButtonVisible =
+      comment.user &&
+      authUser.login === comment.user.login &&
+      !Object.prototype.hasOwnProperty.call(comment, 'repository_url');
```

Comment:
For what this check?

Reply:
Because the description of the issue it is also a commentListItem, so I only show the button if is a comment and when is an issue description it has this field (repository_url), does that make sense?

I think I could extract this `!Object.prototype.hasOwnProperty.call(comment, 'repository_url');`for a function for readability, what do you think?

Reply:
Also I notice that my initial issue post doesn't seem to allow me to edit:

![screen shot 2017-09-22 at 2 17 25 pm](https://user-images.githubusercontent.com/12476932/30760434-2a072cc6-9fa8-11e7-8093-bd19f5eebedb.png)

Not sure if this is where you've set that to happen, but I think it would make sense to show the ActionSheet but _only_ allow for editing for the first issue comment and not deleting. Happy to defer this if you feel like taking care of this in a separate PR however :)

Reply:
I'm using `repository_url` for distinguishing between issue description and issue comment, only issue description has this field like I said before I think is better extract that for a method in order to improve the "understanding".

About editing the description, you are completely right, I didn't realize this possibility, I will take care of that in this PR. 😄